lib/reporter/index.js

```diff
@@ -1,15 +1,7 @@
-'use strict';
+import * as util from './util.js';
 
-const junit = require('./junit');
-const logger = require('./logger');
-const markdown = require('./markdown');
-const tap = require('./tap');
-const util = require('./util');
-
-module.exports = {
-  junit: junit,
-  logger: logger,
-  markdown: markdown,
-  tap: tap,
-  util: util
-};
+export { util };
```

Comment:
Am I missing some syntax to export without importing first?
I tried to do `export * as util from './util.js'` but it doesn't work.

Reply:
I believe the syntax you are looking for is

`export * from './util.js'`

This will export all of the exported object, but not `util` as the default

Reply:
It's not exactly what I want. I want to export the full namespace as `util`.

Reply:
Ahhhh, afaik that isn't possible

/cc @nodejs/modules

Reply:
not possible but was proposed as `export * as` : https://github.com/tc39/ecma262/pull/1174 . I shipped in V8 7.2 (what Node@12 is on) but is not in spec currently.

Reply:
```js
import * as util from './util.js';
export { util };
```

Reply:
Thanks @bmeck. I confirm `export * as` works. I was mislead by VS Code because it doesn't know the syntax.